src/sdg/producten/tests/test_update_catalogs.py

```diff
@@ -1,27 +1,286 @@
+import datetime
+from datetime import date
+
+from django.core.management import call_command
 from django.test import TestCase
 
+from sdg.core.models import ProductenCatalogus
+from sdg.core.tests.factories.catalogus import ProductenCatalogusFactory
+from sdg.organisaties.tests.factories.overheid import LokaleOverheidFactory
+from sdg.producten.models.localized import LocalizedProduct
+from sdg.producten.models.product import Product, ProductVersie
+from sdg.producten.tests.factories.localized import LocalizedProductFactory
+from sdg.producten.tests.factories.product import (
+    ProductVersieFactory,
+    ReferentieProductFactory,
+)
+
 
 class UpdateCatalogTests(TestCase):
+    def setUp(self):
+        super().setUp()
+        self.producten_catalogus_database = ProductenCatalogus.objects.all()
+        self.producten_database = Product.objects.all()
+        self.producten_versies_database = ProductVersie.objects.all()
+        self.localized_producten_database = LocalizedProduct.objects.all()
```

Comment:
I don't really like this. It makes your test a little bit harder to read because you need to look at the setup to see what you are checking/counting. Just add the actual ORM stuff in the test.

plus, it's not a database so the variable names are weird. At most its a list of objects, or "table" - but anyway, just remove.